test/parallel/test-util-inspect.js

```diff
@@ -588,8 +588,9 @@ assert.doesNotThrow(function() {
                      '{ a: 123, inspect: [Function: inspect] }');
 
   const subject = { a: 123, [util.inspect.custom]() { return this; } };
+  const UTC = 'util.inspect.custom';
```

Comment:
@mightyiam It might be best not to give this a name at all but instead spread the string below across multiple lines?